### DISCUSSION TITLE
phase component

### DISCUSSION THREAD

**Discussion by sunhongfu** [facebookresearch/fastMRI#317] [Q&A]
Why does the fastMRI knee dataset has a phase component? I thought it's spin-echo based sequence?
But when I check the raw complex images, it has both real and imaginary components?

**Comment by mmuckley** [comment]
Hello @sunhongfu, the spin-echo won't result in perfect phase cancellation (it's difficult to make a perfect 180-degree pulse).

That being said, I think I do remember substantial phase in the images (well over pi). I've been out of MRI for a few years now, so I don't remember exactly why it is, perhaps due to a phase introduced as part of demodulation? Would be best to ask a sequence person.

**Comment by mmuckley** [comment]
Hello @sunhongfu, I did ask a sequence person and one thing that I forgot about is this: the receive coils themselves have phase. Depending on how coil combination is done, there could be a phase offset from the coils.

Other sources of phase could be RF spoiling or 90-degree offsets that are used in spin echo sequences.
